### ISSUE TITLE
`chlorophyll` slot is not associated with teh Plant Extension!?

### ISSUE THREAD

**Issue by turbomam** [GenomicsStandardsConsortium/mixs#830]
See 
- https://genomicsstandardsconsortium.github.io/mixs/0000177/

Only associated with

- MicrobialMatBiofilm
- MiscellaneousNaturalOrArtificialEnvironment
- Extension
- Sediment
- Water

**Comment by turbomam** [comment]
Maybe this makes sense... in a `PlantAssociated` sample we don't have any uncertainty that plants are involved. But in  water sample we may want to measure chlorophyll as a surrogate for where there are any microscopic green plants present?

**Comment by lschriml** [comment]
Yes, it makes sense where it is used.This is best done by the subject matter experts.It may be frustrating to keep trying to figure this out, and will require a lot of effort, that we could be devoting to our goals.Not sure we need to be devoting effort to reviewing what terms are used in which extensions.This was done already, when they were first developed, and looked at when new checklists/extensions get added.Let’s move forward with finalizing the workflow for new standards development and for our next release.Cheers,Lynn Sent from my iPhoneOn Jul 19, 2024, at 2:40 PM, Mark Andrew Miller ***@***.***> wrote:﻿
Maybe this makes sense... in a PlantAssociated sample we don't have any uncertainty that plants are involved. But in  water sample we may want to measure chlorophyll as a surrogate for where there are any microscopic green plants present?
